### PR TITLE
Update picard to 2.7.1

### PR DESCRIPTION
**Pull request by upendrak**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/picard/2.7.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# https://anaconda.org/BioBuilds/picard
+
+package:
+  name: picard
+  version: "2.7.1"
+
+source:
+  fn: picard-tools-2.7.1.zip
+  url: https://github.com/broadinstitute/picard/archive/2.7.1.zip
+
```
Comment (bgruening) [inline recipes/picard/2.7.1/meta.yaml]: please add a checksum

Comment (upendrak) [inline recipes/picard/2.7.1/meta.yaml]: Ok will do

Comment (upendrak) [inline recipes/picard/2.7.1/meta.yaml]: Done

**Comment by bgruening** [comment]
It seems that the test is not working for this version of picard. Can you run it locally and fix the test?

**Comment by upendrak** [comment]
I tried several other ways to fix the issue but it doesn't work. I want to build it from scratch now. Do you have an example for a java recipe?

**Comment by bgruening** [comment]
please look at: https://github.com/bioconda/bioconda-recipes/pull/3006

**Comment by upendrak** [comment]
Ok thanks. I will wait until you finish building it.

**Pull request closed without merging**